src/components/table/OcTableFiles.vue

```diff
@@ -388,6 +390,50 @@ export default {
 
       return this.$gettext("Select file")
     },
+
+    getSharedWithAvatarDescription(resource) {
+      const resourceType =
+        resource.type === "folder" ? this.$gettext("folder") : this.$gettext("file")
+
+      const userCount = resource.sharedWith.filter(u => !u.link).length
+      const linkCount = resource.sharedWith.filter(u => !!u.link).length
+
+      const baseText = this.$gettext("This %{ resourceType } is shared ")
+      const userText =
+        userCount > 0
+          ? this.$ngettext("with %{ userCount } user", "with %{ userCount } users", userCount)
```

Comment:
Cannot be named `user` / `users`, because it represents users, groups, federated and guest shares.

Reply:
Are we talking variable naming or the "plain text" or both?

Alternative suggestion would be "There are X link shares and Y other shares for this file/folder"?

Reply:
we're talking about the plain text. ;-) `other shares` also doesn't work because that doesn't work standalone.

@pmaier1 @tbsbdr  opinions?

Reply:
We talked yesterday about `This file is shared with 23 recipients` also not being good naming.

Reply:
`This file is shared with X people` (1 person) `and has Y links.` ?

Reply:
@pmaier1 that's exactly the issue. It's also group sharing. So `1 person` / `x people` would be just wrong.

Alternative: make a third split into groups. I.e. `links`, `people` and `groups`.

Reply:
People would include normal user shares, federated shares and guest shares then. Which are all people.

Reply:
Ah, sorry. `This file has X invites and Y links` could work if you don't want to split. Would need to be used consistently in other areas as well.

Reply:
Updated to `"This %{ resourceType } is shared via X invite(s) and Y link(s)"`, hope that gets the message across 😎